pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -415,6 +415,18 @@ message CommandConsumerStatsResponse {
         optional uint64 msgBacklog                  = 15;
 }
 
+message CommandGetTopicsOfNamespace {
+	required uint64 request_id	= 1;
+	required string property    = 2;
+	required string cluster     = 3;
```

Comment:
If this change is going to be merged at 2.0, there might some changes to be coordinated here, because cluster might be removed at 2.0, so that we don't need to include `cluster` here.

this is just a FYI. you don't need to change at this moment. /cc @merlimat

Reply:
Thanks.